src/helpers/axis.jsx

```diff
@@ -1,4 +1,17 @@
 // @flow
+type AxisType =
+  | 'x'
+  | 'y';
```

Comment:
I like this if it is doing what i think it is. It is specifying that the value must be **exactly** `'x'` or `'y'`, right?